### PR TITLE
make guid unique and indexed

### PR DESCRIPTION
**Pull request by mie00**
This is a fix for #63. I am not sure if the extending can be achieved in an easier way than using the \__new__ method.

### CODE REVIEW

**Comment by rkhamis** [comment]
Could be done with a globally defined meta dict.
And whichever class needs to add to it, can copy it and do so

**Comment by mie00** [comment]
meta dict can now be extended or created from scratch, depending on the class implementer's opinion.

File: lib/JumpScale/data/models/Models.py [hunk 1]
```diff
@@ -7,15 +7,31 @@
 
 DB = 'jumpscale_system'
 
+default_meta = {'allow_inheritance': True, "db_alias": DB, 'indexes': ['guid']}
+
+def extend(a, b):
+    print(a,b)
```
Comment (grimpy) [inline lib/JumpScale/data/models/Models.py]: probably this print should be removed?

Comment (mie00) [inline lib/JumpScale/data/models/Models.py]: @grimpy I made a PR

**Pull request merged**